docs/modules/fault-tolerance/pages/fault-tolerance.adoc

```diff
@@ -168,14 +168,7 @@ split-brain protection like this:
 jobConfig.setSplitBrainProtection(true);
 ```
 
-If there’s an even number of members in your cluster, this may mean the
-job will not be able to restart at all if the cluster splits into two
-equally-sized parts. We recommend having an odd number of members.
-
-Note also that you should ensure there is no split-brain condition at
-the moment you are introducing new members to the cluster. If that
-happens, both sub-clusters may grow to more than half of the previous
-size, circumventing the split-brain protection.
+NOTE: Make sure that you disable the split-brain condition when introducing new members to the cluster. Otherwise, both sub-clusters may grow to more than half of the previous size, circumventing split-brain protection.
```

Comment:
```suggestion
NOTE: Make sure that you resolve the split-brain condition before introducing new members to the cluster. Otherwise, both sub-clusters may grow to more than half of the previous size, circumventing split-brain protection.
```

Reply:
Updated as requested. Also removed invisible characters that were preventing H3s from displaying correctly.